tests/update_eve_image/testdata/update_eve_image_oci.txt

```diff
@@ -0,0 +1,50 @@
+# Default EVE version to update
+{{$eve_ver := "6.0.0"}}
+
+# Obtain EVE version from environment variable EVE_VERSION
+{{$env := EdenGetEnv "EVE_VERSION"}}
+
+# If environment variable EVE_VERSION set, use it instead of default
+{{if $env}}{{$eve_ver = $env}}{{end}}
+
+# Obtain eve.hv from config
+{{$eve_hv := EdenConfig "eve.hv"}}
+
+# Obtain eve.arch from config
+{{$eve_arch := EdenConfig "eve.arch"}}
+
+# Combine variables into $short_version
+{{$short_version := printf "%s-%s-%s" $eve_ver $eve_hv $eve_arch}}
+
+# Use eden.lim.test for access Infos with timewait 30m
+{{$test := "test eden.lim.test -test.v -timewait 30m -test.run TestInfo"}}
+
+# Send command to update eveimage from OCI image
+message 'EVE update request'
+eden -t 10m controller edge-node eveimage-update oci://docker.io/lfedge/eve:{{$eve_ver}}-{{$eve_hv}} -m adam://
```

Comment:
I see inside artifacts `checkInstalledVersion: image name not match. config 6.0.0-kvm, image ver 6.0.0-kvm-amd64`. Probably, you should use `short_version` (I mean, add arch) here.

Reply:
Help me understand this. What two values is it finding and mismatched?

Where precisely is it looking for those?

Reply:
You can find them in artifacts of integration GH action: https://github.com/lf-edge/eden/actions/runs/664534549
Direct link: https://github.com/lf-edge/eden/suites/2288088926/artifacts/47914564
Inside archive you will see `info.log` with all info messages received by Adam from Eve.

Reply:
But did that cause a CI error?

> Where precisely is it looking for those?

I didn't mean, where in CI? I meant, what two values is it comparing and finding in error, and why is it different with http?

Reply:
CI just tell, that it cannot find the info we try to find (when we have first item in SwList with defined version and state)
```
InfoContent.dinfo.SwList[0].PartitionState:inprogress|active InfoContent.dinfo.SwList[0].ShortVersion:{{ $short_version }}
```

Reply:
Understood. Time to dig into what and why for it

Reply:
My manual tests worked, so need to figure this part out

Reply:
I see the issue. I need to ask some people. Going to ping them.

Reply:
Never mind, going to fix it in the test.

Reply:
Pushed out, running again.